websites/S/SimulatorFM/presence.ts

```diff
@@ -0,0 +1,63 @@
+const presence = new Presence({
+  clientId: "790721299126943744"
+}),
+browsingStamp = Math.floor(Date.now() / 1000);
+
+let dj: HTMLElement,
+  title: HTMLElement,
+  artist: HTMLElement,
+  player: HTMLAudioElement;
+
+presence.on("UpdateData", async () => {
+  dj = document.querySelector(".m-player__presenter-name");
+  player = document.querySelector(".fa-pause");
+  title = document.querySelector(".m-player__title");
+  artist = document.querySelector(".m-player__artist");
```

Comment:
since the elements arent changed somewhere else just make them const in the updatedata event.

Reply:
Sorted as requested.